doc/app-flows.md

```diff
@@ -0,0 +1,33 @@
+# App Flows
+
+## Main App Flow
+
+1. User receives a text with a redemption link.
```

Comment:
You could mention that deep linking is how to get from step 1 to step 2, and maybe link to the iOS universal links/Android app links sections from the SDK repos.